containers/codespaces-linux/.devcontainer/Dockerfile

```
@@ -151,7 +151,7 @@ RUN apt-get update -yq\
     && sudo -u codespace git config --global core.editor "/home/codespace/.local/bin/git-ed.sh" \
     #
     # Alias code-insiders to code for easy terminal file editing
-    && echo $'\nif [[ $(which code-insiders) && ! $(which code) ]]; then alias code=code-insiders; fi' >> /home/codespace/.bashrc \
+    && echo 'if [[ $(which code-insiders) && ! $(which code) ]]; then alias code=code-insiders; fi' >> /home/codespace/.bashrc \
```

The new line makes the `.bashrc` file look a lot nicer. I think a better fix would be to force it to be executed in a bash shell:
```suggestion
    && /bin/bash -c "echo $'\nif [[ $(which code-insiders) && ! $(which code) ]]; then alias code=code-insiders; fi' >> /home/codespace/.bashrc" \
```

---

Actually, the expansion doesn't work right this way :(. Need to try a different way.